mods/mobs/borers/mob/overrides.dm

```diff
@@ -31,11 +31,10 @@
 		return TRUE
 	return ..()
 
-/obj/item/organ/internal/brain/removed(var/mob/living/user)
+/obj/item/organ/internal/brain/uninstall(in_place, detach, ignore_children)
 	if(istype(owner))
 		var/mob/living/simple_animal/borer/borer = HAS_BRAIN_WORMS(owner)
-		if(borer)
-			borer.detatch()
+		borer?.detatch()
```

Comment:
Previous pattern is preferred, `if(foo) foo.bar()` is conceptually fine if foo isn't truthy, `foo?.bar()` evaluates to just `null` by itself on a line which is odd.

Reply:
I guess I can revert that.

Reply:
done!